web/quiz_views.py

```
@@ -16,8 +16,167 @@
+                            if option_form.cleaned_data and not option_form.cleaned_data.get("DELETE", False):
+                                option = option_form.save(commit=False)
+                                option.question = question
+                                option.order = i
+                                option.save()
+                
+                messages.success(request, "Question added successfully.")
+                
+                # Redirect based on the button clicked
+                if "save_and_add" in request.POST:
+                    return redirect("add_question_specialized", quiz_id=quiz.id)
+                else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)